src/content/self-hosted/install/releases.mdx

```diff
@@ -6,16 +6,52 @@ id: releases
 ---
 
 ## 1.8.0
-TBD
+10 May, 2023
 
 ### Features
+
 - Add support for Kubernetes [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md)
-- Support for custom [labels and annotations](self-hosted/administration/configuration.mdx#namespace) in ingresses managed by Okteto.
+- Support for custom [labels and annotations](self-hosted/administration/configuration.mdx#namespace) in ingresses managed by Okteto
+- Upgraded the Okteto CLI version to [2.15.2](https://github.com/okteto/okteto/releases/tag/2.15.2)
+- Support for custom labels/annotations on every ingress created on dev environments[TBD](link to community)
```

Comment:
@pchico83 i saw at the issue there was a community post comming up for this but havent been able to find it, can you share the link so i can add it here? thnks